src/CogniteSdk.fsproj

```diff
@@ -1,13 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>.NET SDK for Cognite Data Fusion (CDF)</Description>
```

Comment:
We cannot remove this info. Please revert.

Reply:
Looks like Ionide decided to mess with the file. I'll revert it